### PR TITLE
Add Documentation for Wi-Fi Direct

### PR DESCRIPTION
**Pull request by nishadg246**
Created a support page for Wi-Fi Direct broadcasting.

### CODE REVIEW

**Comment by kanodia** [comment]
WHat is Wifi direct? Can you please explain what is it and how it is related to physical web?

File: documentation/WiFi_Direct_Support.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Wi-Fi Direct Support
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: Should this be linked from somewhere else?  (see if we do that for the mdns docs)

File: documentation/WiFi_Direct_Support.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+With Wi-Fi Direct, the Physical Web will allow beacons to broadcast and serve data to users. The advantage this provides is allowing devices without internet connectivity to access data.
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: start off by giving a brief intro to wifi direct, also link to some resource that explains it further.

File: documentation/WiFi_Direct_Support.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Wi-Fi Direct Support
+
+With Wi-Fi Direct, the Physical Web will allow beacons to broadcast and serve data to users. The advantage this provides is allowing devices without internet connectivity to access data. 
+
+Below are instructions on how to setup any capable device to broadcast Physical Web data using Wi-Fi Direct.
+
+To "broadcast", a device must be in a Wi-Fi Direct discoverable state.
+In addition, the device name must follow a defined naming protocl:
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: s/defined naming protocl/naming convention

File: documentation/WiFi_Direct_Support.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+PW-title-port
+
+"title" should be replaced by a fitting title for the content you wish to provide. The port should be where you will be serving HTTP data, whether that be HTML or other types.
+
+In the Android Physical Web App, a feature has been implemented to allow for native sharing of content (currently html, photos, videos, music) over Wi-Fi Direct using the Physical Web.
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: all this should probably be in a different section

File: documentation/WiFi_Direct_Support.md (hunk 1 above)
Comment (nishadg246) [inline documentation/WiFi_Direct_Support.md]: linked it from readme

File: documentation/WiFi_Direct_Support.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+## Wi-Fi Direct Support
+
+[Wi-Fi Direct](http://www.wi-fi.org/discover-wi-fi/wi-fi-direct) allows devices to connect directly rather than through an Access Point. This way, a device can connect and received data through a peer-to-peer connection rather than over the internet. With Wi-Fi Direct, the Physical Web will allow beacons to broadcast and serve data to users. The advantage this provides is allowing devices without internet connectivity to access data. For more informatio
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: informatio?  Sounds Italian.

File: documentation/WiFi_Direct_Support.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+Below are instructions on how to setup any capable device to broadcast Physical Web data using Wi-Fi Direct.
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: I prefer simpler, "Broadcast instructions: "

File: documentation/WiFi_Direct_Support.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+To "broadcast", a device must be in a Wi-Fi Direct discoverable state.
+In addition, the device name must conform to the following naming convention:
+
+PW-title-port
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: Use tic marks and put it on the same line as the previous

File: documentation/WiFi_Direct_Support.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+"title" should be replaced by a fitting title for the content you wish to provide. The port should be where you will be serving HTTP data, whether that be HTML or other types.
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: `<title>` is the title of the content.  `<port>` is the port the HTTP service is running at.

File: documentation/WiFi_Direct_Support.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+### Broadcasting with Wi-Fi Direct
```
Comment (cco3) [inline documentation/WiFi_Direct_Support.md]: File sharing via Wi-Fi Direct and the Physical Web

**Pull request merged**